cpp/src/aztec/join_split_example/proofs/join_split/join_split.test.cpp

```diff
@@ -801,9 +801,9 @@ TEST_F(join_split_tests, test_0_input_notes_and_detect_circuit_change)
     EXPECT_TRUE(result.valid);
 
     // The below part detects any changes in the join-split circuit
-    constexpr uint32_t CIRCUIT_GATE_COUNT = 64000;
+    constexpr uint32_t CIRCUIT_GATE_COUNT = 59175;
```

Comment:
We've changed back to Turbo because the `dsl` module is yet to work with UP. Once that is working (Maxim's on it), we will be doing a fresh PR on bb to switch to Ultra.